### ISSUE TITLE
Dependency conflicts due to older typing-extensions version

### ISSUE THREAD

**Issue by HoodyJazz** [correl/tornado-openapi3#3]
Hi Correl,
we are using your library in our project and it's really helpful, so thanks for the effort. But I am currently running into dependency conflicts with my postgres library, due to the version of typing-extensions you use here. Does it need to be restricted, or could it also be updated to use typing-extensions 4.3.0?

```

[tool.poetry.dependencies]
python = "^3.7"
tornado = "^4 || ^5 || ^6"
openapi-core = "^0.14.2"
ietfparse = "^1.8.0"
typing-extensions = "^4.0.1"

```

Cheers,
Judy

**Comment by cknave** [comment]
It looks like the version conflict was already fixed in ae0b8c09b4ddaf1bf651ec084b45326c6744f8bc. I confirmed the tornado-openapi3 master branch works in a project with typing-extensions 4.4.0, it just needs to be released.

(nudge @correl 👋)

**Comment by correl** [comment]
Apologies, I definitely meant to release that. I've pushed up version 1.1.1 with that change.
